src/components/RecipeForm/index.tsx

```diff
@@ -32,19 +35,17 @@ const RecipeForm = (props: RecipeFormProps): JSX.Element => {
                             options={field.options}
                             id={field.path}
                             gradientOptions={field.gradient_options}
-                            changeHandler={changeHandler}
-                            getCurrentValue={getCurrentValue}
                         />
                     ))}
                 </div>
             )}
             {recipeId && (
                 <Button
-                    onClick={submitPacking}
+                    onClick={onStartPacking}
                     color="primary"
                     variant="filled"
-                    disabled={!submitEnabled}
-                    style={{ width: '100%' }}
+                    disabled={isPacking || !submitEnabled}
```

Comment:
I believe the new `isPacking` removes the need for `submitEnabled`, so I think we could remove `submitEnabled` entirely